README.md

```diff
@@ -56,33 +56,33 @@ Visualizers are estimators (objects that learn from data) whose primary objectiv
 - **Term Frequency**: visualize the frequency distribution of terms in the corpus
 - **TSNE**: use stochastic neighbor embedding to project documents.
 
-And more! Visualizers are being added all the time, be sure to check the examples (or even the develop branch) and feel free to contribute your ideas for Visualizers!
+And more! Visualizers are being added all the time, so be sure to check the examples (or even the develop branch) and feel free to contribute your ideas for Visualizers!
 
 ## Installing Yellowbrick
 
-Yellowbrick is compatible with Python 2.7 or later but it is preferred to use Python 3.5 or later to take full advantage of all functionality. Yellowbrick also depends on Scikit-Learn 0.18 or later and Matplotlib 1.5 or later. The simplest way to install Yellowbrick is from PyPI with pip, Python's preferred package installer.
+Yellowbrick is compatible with Python 2.7 or later but it is preferred to use Python 3.5 or later to take full advantage of all functionality. Yellowbrick also depends on scikit-learn 0.18 or later and matplotlib 1.5 or later. The simplest way to install Yellowbrick is from PyPI with pip, Python's preferred package installer.
 
     $ pip install yellowbrick
 
 Note that Yellowbrick is an active project and routinely publishes new releases with more visualizers and updates. In order to upgrade Yellowbrick to the latest version, use pip as follows.
 
-    $ pip install -u yellowbrick
+    $ pip install -U yellowbrick
 
-You can also use the `-u` flag to update Scikit-Learn, matplotlib, or any other third party utilities that work well with Yellowbrick to their latest versions.
+You can also use the `-U` flag to update scikit-learn, matplotlib, or any other third party utilities that work well with Yellowbrick to their latest versions.
 
-If you're using Windows or Anaconda, you can take advantage of the conda utility to install Yellowbrick:
```

Comment:
I deleted "Windows or" since I thought the wording implied that Anaconda was natively installed on Windows.

Reply:
I think that's a good edit, but perhaps we could say the following:

```
If you're using Anaconda (recommended for Windows users), you can take advantage of the `conda` utility to install Yellowbrick: 
``

What do you think?